bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/MD5Hashes.java

```diff
@@ -24,17 +24,17 @@
  */
 public final class MD5Hashes implements Hashes {
 
-	private final String algorithm = "MD5"; //$NON-NLS-1$
+	private static final String ALGORITHM = "MD5"; //$NON-NLS-1$
```

Comment:
We never use `static` methods/fields unless it is proven to be necessary (evident performance issue, for example) or unless for something that has indeed static nature (usually is it some restriction out of the perimiter of the functional core - no occurencies in this project). 

In cases like this `static` is used to _optimize_ class runtime, hurting readability of clear class sense, which is always a bad idea (in both dimentions) and has out-of-scope side effect: classes, and thus everithing static, are served by separate memory part, which functions according to its own rules.

Reply:
As you prefer. I just changed it because I noticed the Sonar-Lint complaint and usually try to make common 'things' static, but since it violated your preference I've reverted it.